### ISSUE TITLE
email setting/sending not working

### ISSUE THREAD

**Issue by rbwilkinson** [Orangescrum/orangescrum#110]
I would prefer to use my own email system that shares the same server.
More in depth documentation is sorely needed.

However I cannot even get GMAIL to work.
Do I need to configure any settings on gmail.com?

The email test gives :
php_openssl in php.ini is enabled 
smtp.gmail.com:465 () is open.
Simple Email Caught exception: Invalid email: "emailId"
Template Email Caught exception: Invalid email: "emailId"

Help?

**Comment by ajobbins** [comment]
Agreed. I've put in my SendGrid account details, but when I try and add a user I get a popup saying:

**Comment by ajobbins** [comment]
Agreed. I've put in my SendGrid account details, but when I try and add a user I get a popup saying:

Make sure that you have done the below required changes in `app/Config/constants.php`
You have provided the details of SMTP email sending options in `app/Config/constants.php`
You have updated FROM_EMAIL_NOTIFY and SUPPORT_EMAIL in `app/Config/constants.php`

And yes, I've done all those things.

**Comment by Prabhudas** [comment]
Hi ,

Can you please share the constants.php file?
